dgraph/cmd/alpha/dashboard.go

```diff
@@ -39,8 +39,8 @@ func homeHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	dgraph := "\nDgraph is up and running...\n\n"
-	dgraph = dgraph + "Now you can access your cluster via Ratel Dashboard. Remember, the PORT to use there to connect to Alpha is 8080 + offset if the case.\n"
+	dgraph := "\nDgraph is up and running...\n\n" // swiftlint:disable:next line_length
+	dgraph = dgraph + `Now you can access your cluster via Ratel Dashboard.Remember, the PORT to use there to connect to Alpha is 8080 + offset if the case.\n`
```

Comment:
This all should be refactored into one statement. For instance:

```golang
dgraph := `Dgraph is up and running...

Now you can access...
`

```

Reply:
Yeah, I know. But when printing it adds spaces. And it looks messy visually.

Reply:
I have to do something like this
```
	dgraph = dgraph + `Now you can access your cluster via Ratel Dashboard.
Remember, the PORT to use there to connect to Alpha is 8080 + offset if the case.\n`
```

Reply:
Have a look at this: https://go.dev/play/p/vAqBiZZ0tYT

Reply:
Also, I don't think you should call out what port to use. It might have been mapped to some other port. You could examine the http.Request.Host parameter and echo that back to the user. Or, I'd say leave it out entirely.

Reply:
I'm not in favor of using backtick here. Cuz if I need a concise spacing, distancing. I can't. If it would be HTML/CSS. Would be nice. But this is go printing.

I finally have found a nice way of doing it.

Thank you Matthew. Your comments helped me to find another way hehehe

About the port. It is just a tipo. Anyone who changes it, should be at a level that this tip is not necessary. Right? If you by conscience change the offset of the port. You know what you are doing.

Reply:
Did you try running that go.dev link that I supplied? It formats everything as it is formatted between the ticks. I'm pretty sure it'll format the same way to the Writer and onto to the browser window. Are you seeing something different?

I can't get on board with adding extra machine instructions to simply concat a static string.

Reply:
Yes. It does. it is "the same". More clean in a way - that was my first attempt. But the lines need to be aligned to left in the code. And if you need to add more lines. You will place literal lines inside the block. With concat lines with quoted strings the fmt adds 2 tabs. It gets more "compact".

Both ways result in the same result. But I'd prefer without backticks.

Reply:
So a good practice regarding "what style should I use?" when something like this comes up is to look at the go standard library and see what's done there. Here's one of many places there where long string literals are encoded as I recommended: https://github.com/golang/go/blob/bb917bd1b212dc8fff3852fa164667cd06b9f653/src/cmd/vendor/github.com/google/pprof/internal/driver/cli.go#L304

Reply:
OK fine. But the "if" should be capitalized.

Reply:
Done.